__tests__/nc_news.test.js

```diff
@@ -117,3 +118,55 @@ describe('GET /api/articles/:article_id', () => {
 		});
 	});
 });
+
+describe('GET /api/articles/:article_id/comments', () => {
+	describe('Endpoint Behaviour', () => {
+		test('GET:200 expects correct status code', () => {
+			return request(app).get('/api/articles/1/comments').expect(200);
+		});
+		test('GET:200 expects correct comment format & correct quantity', () => {
+			return request(app)
+				.get('/api/articles/1/comments')
+				.then(({ body }) => {
+					const comments = body.comments;
+					expect(comments).toHaveLength(11);
+					comments.forEach((comment) => {
+						expect(typeof comment.comment_id).toBe('number');
+						expect(typeof comment.votes).toBe('number');
+						expect(typeof comment.created_at).toBe('string');
+						expect(typeof comment.author).toBe('string');
+						expect(typeof comment.body).toBe('string');
+						expect(typeof comment.article_id).toBe('number');
+					});
+				});
+		});
+		test('GET:200 expects correct ordered by default from newest ASC', () => {
```

Comment:
Is the first comment gonna be the newest when sorting them with ascending order?